docs/notebooks/delayed-reward.ipynb

```diff
@@ -508,12 +684,61 @@
                 }
             ],
             "source": [
-                "import matplotlib.pyplot as plt\n",
+                "np.random.seed(3)\n",
+                "optimal_decay = study.best_params['decay_rate']\n",
+                "optimal_decay_regret = run_decay_simulation(decay_rate=optimal_decay, oracle=oracle)\n",
                 "\n",
-                "plt.plot(np.cumsum(regret))\n",
+                "plt.plot(np.cumsum(optimal_decay_regret))\n",
                 "plt.xlabel(\"Round\")\n",
-                "plt.ylabel(\"Cumulative Regret\")"
+                "plt.ylabel(\"Cumulative Regret\")\n",
+                "plt.title(f\"Cumulative regret for optimal decay {optimal_decay:0.3f} : {np.cumsum(optimal_decay_regret)[-1]:0.1f}\")\n",
+                "summary_of_regrets['Regret with optimal decay rate'] = np.cumsum(optimal_decay_regret)[-1]"
             ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Bellow is a table summary of the four different scenarios / strategies that we tried.\n",
+                "We could get better results with the optimal decay rate than without decay.\n",
+                "\n",
+                "Note that there is a lot of variation between runs, so we should run multiple runs with each strategy to verify how the distribution of the results work."
```

Comment:
I'd word this a bit more strongly, even. Something like how the `decay_rate` is yet another hyperparameter to pick and should be chosen with the same care and deliberation as any other hyperparameter. Perhaps showing the cumulative regret associated with one of the "worst" values Optuna tried may help.

Reply:
Added a worst case decay_rate